pkg/testutil/testutil.go

```diff
@@ -560,6 +562,10 @@ func M(m *testing.M) {
 	flag.BoolVar(&flagTestKillDaemon, "test.allow-kill-daemon", false, "enable tests that kill the daemon")
 	flag.BoolVar(&flagTestIPv6, "test.only-ipv6", false, "enable tests on IPv6")
 	flag.BoolVar(&flagTestKube, "test.only-kubernetes", false, "enable tests on Kubernetes")
+	flag.BoolVar(&flagTestFlaky, "test.only-flaky", false, "enable testing of flaky tests only")
```

Comment:
Is there any way to rather skip flaky tests

Reply:
That is the default behavior.
If you do not set `test.only-flaky`, only the `non-flaky` tests tests will run.

Reply:
Next PR will cleanup the github actions and separate flaky (with retry) and non-flaky (no retry)

Reply:
Could you document it in -help

Reply:
Will do.

Reply:
Done.